packages/runtime/runtime-definitions/src/runtimeLayerCompatFeatureNames.ts

```diff
@@ -0,0 +1,12 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * This feature indicates the ContainerRuntime will encode handles
+ * If the Runtime layer supports this feature, the DataStore layer need not encode handles (but do bind them)
+ *
+ * @internal
+ */
+export const encodeHandlesInContainerRuntime = "encodeHandlesInContainerRuntime";
```

Comment:
```suggestion
export const encodeHandlesInContainerRuntime = "encodeHandlesInContainerRuntime" as const;
```

Reply:
Yeah I was going to suggest to @agarwal-navin to revisit the conventions I'm setting in here.  I'm wondering about doing it like this actually:

```
export const layerCompatFeatures = {
  encodeHandlesInContainerRuntime = "encodeHandlesInContainerRuntime",
} as const;
```

So usages are more self-documenting:  `layerCompatFeatures.encodeHandlesInContainerRuntime`

Maybe work in more specific naming to indicate which layers (and direction if applicable) it's for.